src/Contract/Address.purs

```diff
@@ -4,7 +4,9 @@ module Contract.Address
   , enterpriseAddressStakeValidatorHash
   , enterpriseAddressValidatorHash
   , getNetworkId
+  , addressWithNetworkTagFromBech32
   , addressWithNetworkTagToBech32
+  , addressFromBech32'
```

Comment:
If I try to export `addressFromBech32` I get an `ExportConflict` error (Export for value Serialization.Address.addressFromBech32 conflicts with value Contract.Address.addressFromBech32) which I find strange as I don't export but import `Serialization.Address.addressFromBech32`. Anyway, should I just give this function another name or is there a workaround?

Reply:
It is exported, via the `module SerializationAddress` export. I'd argue the one from `SerializationAddress` shouldn't be exported given the new functions and can replace it. @klntsky what do you think?

Reply:
thanks for the tip, I removed `addressFromBech32` from `module SerializationAddress`. however, the new `addressFromBech32` changes the API as it returns a `Contract r (Maybe Address)` instead of a `Maybe Address`. a problem?

Reply:
We'll need to have the changelog reflect under Changes that `addressFromBech32` is lifted to `Contract` now.

@ngua What do you think the migration strategy should be? I think for users to use the old one, they had to dip into internal functions anyway to get from CSL address to plutus address, so really it's unsupported to use the old `addressFromBech32`?